tests/settings/ConfigurationGatheringTest.py

```diff
@@ -40,20 +45,19 @@ def test_gather_configuration(self):
             sections, local_bears, global_bears, targets = (
                 gather_configuration(
                     *args,
-                    arg_list=["-S",
-                              "test=5",
-                              "-c",
-                              escape(temporary, "\\"),
-                              "-s"]))
```

Comment:
you could just add ` + self.min_args` here, and not break blame on the above lines.
(same applies elsewhere below, where the line length isnt going to be a problem)
Up to you